distribution/springboot_setup/README.md

```diff
@@ -50,10 +50,10 @@ By default, the exchange of data between the sender and the receiver is done wit
 
 Copy and rename the _application.properties_ files:
 ```
-cp distribution/springboot_setup/sender/application.properties app/src/main/resources/application-sender.properties
+cp distribution/springboot_setup/sender/_application.properties app/src/main/resources/application-sender.properties
```

Comment:
These files were rename already in master

Reply:
Yes, unfortunately, this was not reflecting in the README.

Reply:
i see